utils/python/tests/scripts_regression_tests.py

```diff
@@ -170,44 +170,78 @@ def setup_proxy():
 ###############################################################################
 class J_TestCreateNewcase(unittest.TestCase):
 ###############################################################################
-    def setUp(self):
-        self._testroot = MACHINE.get_value("CIME_OUTPUT_ROOT")
+    @classmethod
```

Comment:
I wanted the case from the create_newcase test left around so that I could reuse it for the create_clone tests - this is the solution I found.   Now the setup is only run at the beginning of the class and teardown only happens after all the class tests are complete.   I think that this is something we can build further on.